org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/WListbox.java

```diff
@@ -1255,5 +1277,100 @@ public void setAllowIDColumnForReadWrite(boolean allowIDColumnForReadWrite)
 	{
 		this.allowIDColumnForReadWrite = allowIDColumnForReadWrite;
 	}
+	
+	   /**
+     * Retrieve the width of the column
+     * 
+     * @param columnName the name of the column to find the width of.
+     * @param columnList list of the columns and their width
+     * @return the width of the column
+     */
+	private String getColumnWidth(String columnName, List<String>columnList) 
+	{
+		String width ="null";
+		String[] w;
+		
+		for (String column : columnList)
+		{
+			w = column.trim().split("=");
+			if (w[0].equals(columnName))
+			{
+				return w[1];
+			}
+		}
+		return width;
+	}
+	
+    /**
+     * Render the header of the WListbox with the columns width of the saved column width
+     * 
+     */
+	public void renderHeaderColumnWidth () 
+	{
+		if (wListBoxName != null && getListHead() != null) 
+		{
+			MWlistboxCustomization WListBoxCustomization =    MWlistboxCustomization.get(Env.getCtx(), Env.getAD_User_ID(Env.getCtx()), wListBoxName, null);
+			Boolean isHasCustomizeData =  WListBoxCustomization != null &&
+					  WListBoxCustomization.getAD_Wlistbox_Customization_ID() > 0 &&
+					  WListBoxCustomization.getCustom() != null &&
+					  WListBoxCustomization.getCustom().trim().length() > 0; 			
+			if (isHasCustomizeData) 
+			{
+				String[] custom = WListBoxCustomization.getCustom().trim().split(",");		
+				List<String>columnList = Arrays.asList(custom);
+				ColumnInfo[] Columns = this.getLayout();
+				Listhead listHead =  getListhead();
+				if (listHead != null && columnList.size() > 0) 
+				{
+					List<?> headers = listHead.getChildren();
+					int i = 0;
+					for(Object obj : headers)
+					{
+						Listheader header = (Listheader) obj;
+						String columnWidth = this.getColumnWidth(Columns[i].getColumnName(), columnList);
+						if (!("null".equals(columnWidth)) || !Util.isEmpty(columnWidth))	
```

Comment:
shouldn't this be "&&" instead ? also, the "&&" or "||" is not needed if the getColumnWidth(...) method just return null instead.

Reply:
You are right.